client/reducers/locBarReducer.js

```diff
@@ -1,10 +1,10 @@
-export default function locBar(state = '', action) {
+export default function locBar(state = [[]], action) {
   switch (action.type) {
     case 'Location_Search':
       console.log('inside the locBarReducer');
       console.log('action: ', action);
       console.log(action.payload.data);
-      return { ...state, GoogleResults: action.payload.data };
+      return [action.payload.data]
```

Comment:
the reducer has to return a copy of the existing state and append any extra data to it.
Only returning the payload data as an array will cause issues later on. You will have to access the payload data differently, as a property on the redux state obj, but it should still be available for you to use